server/src/main/java/org/elasticsearch/index/mapper/GeoPointFieldMapper.java

```diff
@@ -177,13 +220,23 @@ public FieldMapper build(MapperBuilderContext context) {
     private static final Version MINIMUM_COMPATIBILITY_VERSION = Version.fromString("5.0.0");
 
     public static TypeParser PARSER = new TypeParser(
-        (n, c) -> new Builder(n, c.scriptCompiler(), IGNORE_MALFORMED_SETTING.get(c.getSettings()), c.indexVersionCreated()),
+        (n, c) -> new Builder(
+            n,
+            c.scriptCompiler(),
+            IGNORE_MALFORMED_SETTING.get(c.getSettings()),
+            c.indexVersionCreated(),
+            c.getIndexSettings().getMode()
+        ),
         MINIMUM_COMPATIBILITY_VERSION
     );
 
     private final Builder builder;
     private final FieldValues<GeoPoint> scriptValues;
     private final Version indexCreatedVersion;
+    /** The metric type (gauge, counter, summary) if  field is a time series metric */
+    private final TimeSeriesParams.MetricType metricType;
```

Comment:
The comment here looks not relevant?

Reply:
Pretty sure I copied this field, and the comment line, from other mappers. But I can remove the comment, as I agree the value is not high.

Reply:
The only relevance is that setting this field to null is the marker that this is not a time-series metric field.

Reply:
I guess for GeoPoin it can only be a position metric?

Reply:
Yes, but it can also be null, indicating that it is not a time-series metric at all. So we could just use a boolean, but that creates some asymmetry with all the other field-mappers.

Reply:
I just deleted the comment.